core/services/ocr2/delegate.go

```diff
@@ -125,6 +128,18 @@ func (d Delegate) ServicesForSpec(jobSpec job.Job) ([]job.ServiceCtx, error) {
 			return nil, errors.Wrap(err2, "get chainset")
 		}
 
+		var sendingKeys []string
+		ethSendingKeys, err2 := d.ethKs.GetAll()
+		if err2 != nil {
+			return nil, errors.Wrap(err2, "get eth sending keys")
+		}
+
+		// Automatically provide the node's local sending keys to the job spec.
+		for _, s := range ethSendingKeys {
+			sendingKeys = append(sendingKeys, s.Address.String())
+		}
+		spec.RelayConfig["sendingKeys"] = sendingKeys
```

Comment:
Where do these updates to the relay config end up getting persisted? I see another update on line 154 but not sure where this ends up getting saved to the DB.

Reply:
I think it is used at runtime and not saved in the DB. @essamhassan is that correct?